front-end/src/host/HostQuestionnaireView.tsx

```diff
@@ -120,35 +121,7 @@ export default function HostQuestionnaireView(
             >
               {waitingPlayers.map((name: string, i: number) => (
                 <li className="li" key={i}>
-                  <Paper
-                    elevation={3}
-                    sx={{
-                      "&:hover": {
-                        cursor: "pointer",
-                        boxShadow: 8,
-                        textDecoration: "line-through",
-                      },
-                      background: "linear-gradient(-45deg, cyan, magenta)",
-                      borderRadius: "20px",
-                      width: "15vw",
-                      margin: "auto",
-                      marginBottom: "1.5vh",
-                    }}
-                    onClick={() => onPlayerKick(name)}
-                  >
-                    <p
-                      style={{
-                        margin: 0,
-                        fontFamily: "Concert One",
-                        color: "White",
-                        paddingTop: "3px",
-                        paddingBottom: "3px",
-                        fontSize: "1.5em",
-                      }}
-                    >
-                      {name}
-                    </p>
-                  </Paper>
+                  <PlayerBadge name={name} onClick={() => onPlayerKick(name)} />
```

Comment:
`<PlayerBadge />` component can be reused every time we need something that looks like this